modules/bound-sa-tokens-configuring.adoc

```diff
@@ -23,15 +23,7 @@ This step is typically not required if the bound tokens are used only within the
 ====
 If you update the `serviceAccountIssuer` field and there are bound tokens already in use, all bound tokens with the previous issuer value will be invalidated. Unless the holder of a bound token has explicit support for a change in issuer, the holder will not request a new bound token until pods have been restarted.
 
-If necessary, you can use the following command to manually restart all pods in the cluster. Be aware that running this command causes a service interruption, because it deletes every running pod in every namespace. These pods will automatically restart after they are deleted.
-
-[source,terminal]
-----
-$ for I in $(oc get ns -o jsonpath='{range .items[*]} {.metadata.name}{"\n"} {end}'); \
-      do oc delete pods --all -n $I; \
-      sleep 1; \
-      done
-----
+If necessary, you can manually restart all pods in the cluster so that the holder will request a new bound token. Before doing so, make sure to wait for a new revision of the Kubernetes API server pods to roll out with your service account issuer changes.
```

Comment:
Change 'make sure' to 'verify' (Red Hat SSG -  https://redhat-documentation.github.io/supplementary-style-guide/#_m)